pkg/errors/errors.go

```diff
@@ -61,3 +61,18 @@ func NewEgressURLError(url string) error {
 		message:   fmt.Sprintf("egressURL error: %s", url),
 	}
 }
+
+type KmsError struct {
+	message string
+}
+
+// Ensure KmsError implements the error interface
+var _ error = &KmsError{}
```

Comment:
This just tries to create a variable of type `error` from our `KmsError`, and if that fails, compilation would fail. This is just a nice way of ensuring you implemented the interface `errror`